client-lib/src/main/java/com/ibm/ws/repository/resources/internal/RepositoryResourceImpl.java

```diff
@@ -1677,11 +1685,11 @@ public InputStream getInputStream() throws RepositoryBackendException, Repositor
             try {
                 return _client.getAttachment(_asset, _attachment);
             } catch (IOException e) {
-                throw new RepositoryBackendIOException("Failed to get read attachment", e, _repoConnection);
+                throw new RepositoryBackendIOException("Failed to get read attachment", e, getRepositoryConnection());
```

Comment:
This looks odd. In the constructor of an exception we are calling a method that may well throw an exception. That would be hugely confusing if it happened